src/modules/GroupChannel/components/FileViewer/index.scss

```diff
@@ -4,6 +4,10 @@
  */
 @import '../../../../styles/variables';
 
+:root {
+  --file-viewer-header-button-width: 64;
+}
```

Comment:
https://github.com/sendbird/sendbird-uikit-react/blob/main/src/styles/_variables.scss

It seems there's another way to define the CSS variables